UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/css/maritime-safety-information.scss

```diff
@@ -11,4 +11,15 @@ a, button{
         @extend %focus;
     }
 }
+.form-label {
+    margin-bottom: 0.5rem !important;
```

Comment:
Please confirm with Mubeen, if both css and scss files with same changes needs to be included in commit?